app/src/main/java/org/jellyfin/androidtv/TvApp.java

```diff
@@ -94,7 +100,14 @@ protected void attachBaseContext(Context base) {
     public void onCreate() {
         super.onCreate();
         app = (TvApp) getApplicationContext();
-        playbackManager = new PlaybackManager(new AndroidDevice(this), new AndroidLogger("PlaybackManager"));
+        JellyfinOptions.Builder options = new JellyfinOptions.Builder();
+        JellyfinAndroidKt.android(options, this);
+        options.setLogger(new AndroidLogger());
```

Comment:
Why bother keeping the logger if we're removing the credential management?

Reply:
Some other places currently use the logger